src/index.js

```diff
@@ -90,3 +90,4 @@ displayTaskFromStore();
 editFunction();
 
 tasksMenu();
+completedTask();
```

Comment:
- [ ] Clear all completed button is not working. Kindly fix it. [Requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=Implement%20a%20function%20for%20the%20%22Clear%20all%20completed%22%20button%20(use%20filter()%20method).)

Reply:
@ahmetbozaci 
Thank you so much for the review. The clear all completed button is working from my side. After pressing the clear all completed button, you need to click on the refresh button. All the checked tasks are then cleared.

Reply:
@Sonickmumba I can't click on the clear all button. But if it is working on your side, can you send it for review again? The next code reviewer will check.

Reply:
- Please note that when the clear all completed button is clicked, it only rules over the task without actually clearing them. As suggested by the previous reviewer kindly fix this 👍🏽